shared/tsconfig.json

```diff
@@ -8,7 +8,14 @@
         "jsx": "preserve",
         "incremental": true,
         "outDir": "./.tsOut/emit",
-        "tsBuildInfoFile": "./.tsOut/cache"
+        "allowSyntheticDefaultImports": true,
+        "tsBuildInfoFile": "./.tsOut/cache",
+        "baseUrl": ".",
+        "paths": {
```

Comment:
The rn-fetch-blob types seem broken. This is a workaround to override that type. It looks like this commit is wrong (https://github.com/joltup/rn-fetch-blob/commit/8bc5cebddaddc2e6d6d27dcb248d2e9bb24aecaf)